src/sections/wallet/assets/hydraPositions/data/WalletAssetsHydraPositionsData.utils.tsx

```diff
@@ -66,67 +71,98 @@ export const useHydraPositionsData = () => {
 
     const rows: HydraPositionsTableData[] = positions.data
       .map((position) => {
-        const meta = metas.data.find(
-          (m) => m.id.toString() === position.assetId.toString(),
+        const assetId = position.assetId.toString()
+        const meta = metas.data.find((m) => m.id.toString() === assetId)
+        const details = detailsList.data?.find(
+          (d) => d.id.toString() === assetId,
         )
         const lrnaMeta = metas.data.find(
           (m) => m.id.toString() === apiIds.data.lrnaId,
         )
         const omnipoolAsset = omnipoolAssets.data.find(
-          (a) => a.id.toString() === position.assetId.toString(),
+          (a) => a.id.toString() === assetId,
         )
         const omnipoolBalance = omnipoolBalances.find(
-          (b) => b.data?.assetId.toString() === position.assetId.toString(),
-        )
-
-        if (
-          !meta ||
-          !lrnaMeta ||
-          !omnipoolAsset?.data ||
-          !omnipoolBalance?.data
+          (b) => b.data?.assetId.toString() === assetId,
         )
-          return null
 
-        const id = position.assetId.toString()
-        const symbol = meta.symbol
-        const name = getAssetName(meta.symbol)
+        const symbol = meta?.symbol || "N/A"
+        const name = details?.name || getAssetName(meta?.symbol)
 
         const [nom, denom] = position.price.map((n) => new BN(n.toString()))
         const price = nom.div(denom)
         const positionPrice = price.times(BN_10.pow(18))
 
-        const params: Parameters<typeof calculate_liquidity_out> = [
-          omnipoolBalance.data.balance.toString(),
-          omnipoolAsset.data.hubReserve.toString(),
-          omnipoolAsset.data.shares.toString(),
-          position.amount.toString(),
-          position.shares.toString(),
-          positionPrice.toFixed(),
-          position.shares.toString(),
-        ]
-        const lernaOutResult = calculate_liquidity_lrna_out.apply(this, params)
-        const liquidityOutResult = calculate_liquidity_out.apply(this, params)
-        if (liquidityOutResult === "-1" || lernaOutResult === "-1") return null
+        let lernaOutResult = "-1"
+        let liquidityOutResult = "-1"
+
+        if (omnipoolBalance?.data && omnipoolAsset?.data) {
+          const params: Parameters<typeof calculate_liquidity_out> = [
+            omnipoolBalance.data.balance.toString(),
+            omnipoolAsset.data.hubReserve.toString(),
+            omnipoolAsset.data.shares.toString(),
+            position.amount.toString(),
+            position.shares.toString(),
+            positionPrice.toFixed(0),
+            position.shares.toString(),
+          ]
+          lernaOutResult = calculate_liquidity_lrna_out.apply(this, params)
+          liquidityOutResult = calculate_liquidity_out.apply(this, params)
+
+          // console.table([
```

Comment:
Musi tady byt console log?

Reply:
nechal jsem ho tady, casto toty veci debuguju a musim tam furt ty logy pridavat, tak to jen pro budoucnost necham zakomentovane